### DISCUSSION TITLE
Upgrade guide from version 2.0.4 to 4.0.0 ?

### DISCUSSION THREAD

**Discussion by franjorub** [react-native-elements/react-native-elements#3340] [Q&A]
Hi, is there a changelog or post to upgrade from those versions?

**Comment by arpitBhalla** [comment]
Hey, I think this might help,
https://github.com/react-native-elements/react-native-elements/blob/next/CHANGELOG.old.md

**Reply by Nantris** [reply]
Is there any upgrade guide from 3.x to 4.x?

**Reply by arpitBhalla** [reply]
https://reactnativeelements.com/blog/2022/05/15/rneui-migration-guide

**Reply by Nantris** [reply]
Thanks for the link! I found it pretty easy to upgrade, but I was surprised to read that `ThemeProvider` should be used even if you use the default theme. If that's not mentioned somewhere in the docs already, it probably should be.
